packages/ui/src/components/molecules/Topbar.tsx

```diff
@@ -83,6 +88,12 @@ export const Topbar: FC<ITopbar> = ({
     error: <SyncProblem fill={theme?.palette.warn.main} />,
   };
 
+  const TitleStyle = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 16px;
+  `;
```

Comment:
Move this out of the component to solve this error 
![image](https://github.com/Cypherock/cypherock-cysync/assets/80461749/ecb6caa8-9ca5-4544-bff9-7590cdf891ec)